packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -548,6 +693,8 @@ class Scaffold extends StatefulWidget {
 /// the current [BuildContext] using [Scaffold.of].
 class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
 
+  _ScaffoldGeometryNotifier _geometryNotifier;
```

Comment:
we should put this in the section of this class that uses it, i.e. below the `// INTERNALS` comment above initState

Reply:
done